### PR TITLE
Feat/history

### LINKED ISSUE #8: Choose from history
When I started to youse goproxie, non-interactive mode is pretty nice, but I also found my self building aliases :joy: 

```sh
alias proxie-xxx="goproxie -project=xxx -proxy_type=xxx -cluster=xxx -pod=xxx"
```

Goproxie could store those calls and provide select interface with search, e.g.

```
$ goproxie history
> -project=xxx -proxy_type=xxx -cluster=xxx -pod=xxx
> -project=yyy -proxy_type=xxx -cluster=yyy -pod=xxx
```

What do you think of this?

### PR DESCRIPTION
**Pull request by smoliji**
- adds persisted configuration
- adds history sub-command using `goproxie history`
- adds `no-save` flag for the history feature to avoid duplicates when using history

Fixes https://github.com/AckeeCZ/goproxie/issues/8

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -229,13 +234,19 @@ func readArguments() {
...
+	flags.noSave = flag.Bool("no-save", false, "Don't save invocation to history")
 	flag.Parse()
 	gcloud.SetGcloudPath(*gcloudPath)
 	kubectl.SetKubectlPath(*kubectlPath)
 }
 
 func main() {
 	readArguments()
+	store.Initialize()
+	if os.Args[1] == "history" {
```
Comment (sandratatarevicova) [inline main.go]: What if there are no arguments?

File: internal/history/history.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+const KeyCommands = "history.commands"
+
+func extractPodBaseName(pod *kubectl.Pod) string {
+	// TODO
+	return pod.Name
+}
+
+func StorePodProxy(projectId string, cluster *gcloud.Cluster, namespace string, pod *kubectl.Pod, localPort int, remotePort int) {
+
+	record := fmt.Sprintf("-project=%v -cluster=%v -namespace=%v -pod=%v -local_port=%v -proxy_type=pod", projectId, cluster.Name, namespace, extractPodBaseName(pod), localPort)
```
Comment (sandratatarevicova) [inline internal/history/history.go:25]: What about the remote port? Do we want to set it as well?

Comment (smoliji) [reply internal/history/history.go:25]: We do, but this have to be resolved first https://github.com/AckeeCZ/goproxie/issues/6

File: main.go (hunk 1 above)
Comment (smoliji) [reply main.go]: Tru. Fixed.

**Pull request merged**